feat(top recipes section): recipe Link card added with links to some …

Greimil opened the pull request:
…recipes

Greimil committed:
feat(top recipes section): recipe Link card added with links to some recipes

joekotvas commented on src/features/HomePage/components/Top-recipes/topRecipes.jsx:
```diff
@@ -1,27 +1,39 @@
-        alt="Fruit Kabobs"
-        className="col-start-1 col-end-4 md:col-start-1 md:col-end-4  lg:col-start-1 lg:col-end-2 lg:row-start-4 mx-auto"
+      <RecipeLinkCard
+        text="ChocoPB Chip"
```
Maybe call them "Choco PB Chip Banana Muffins"?

joekotvas commented on src/features/HomePage/components/Top-recipes/topRecipes.jsx:
```diff
@@ -1,27 +1,39 @@
+      <RecipeLinkCard
+        text="Chicken Wings"
+        src={ChickenWings}
+        bgColor="bg-[#FF7D16]"
```
Here you could use the gradients (ex. bg-gradient-tangerine-diagonal).

joekotvas commented on src/features/ui/RecipeLinkCard/RecipeLinkCard.jsx:
```diff
@@ -0,0 +1,37 @@
+      >
+        <div className="white-circle w-24 h-24 bg-white rounded-full flex justify-center items-center ">
+          {" "}
+          <img src={src} alt={alt} className="w-[50px]" />{" "}
```
The images look better if you make them bigger, say 5rem (w-20).

joekotvas commented on src/features/ui/RecipeLinkCard/RecipeLinkCard.jsx:
```diff
@@ -0,0 +1,37 @@
+      <div
+        className={`absolute ${bgColor} h-24 w-52 top-4 right-0 rounded-tr-full rounded-br-full flex justify-center items-center  `}
+      >
+        <h4 className="font-[open-sans] text-center text w-36 font-bold">
```
Here you will want to use 
`<Heading level="h4" className="text-[#ffffff]">
{text}
</Heading>`

joekotvas replied:
I'll be adding a pull request soon that will configure h4 to use Open Sans by default.

joekotvas reviewed (changes requested):
I love what you've done! Here are some notes.

Greimil committed:
style(recipe link card): recipe Link Card Component styling, typography and bgColor change

Greimil committed:
style: recipeLinkCard typography, img size and bg color change

Greimil committed:
Merge branch 'development' into CH-61-make-the-start-exploring-button-navigate-to-search-page

joekotvas committed:
Fix: Add missing closing parenthesis and semicolon

Add missing `);` that was causing the build to fail

joekotvas committed:
fix(home page): tune home page responsive styles

A plethora of adjustments to fix issues such as the background color for the search section
disappearing on mobile, the top recipe card font not displaying correctly, top recipe cards
overlapping on medium sized screens, etc.

joekotvas committed:
style(global.css): remove empty line

joekotvas committed:
fix(top recipe cards): ensure font size is small enough to fit on two lines
